src/DynamoCoreWpf/ViewModels/Menu/PreferencesViewModel.cs

```diff
@@ -141,17 +141,6 @@ public string SavedChangesTooltip
             }
         }
 
-        /// <summary>
-        /// Returns the state of the Preferences Window Debug Mode
-        /// </summary>
-        public bool PreferencesDebugMode
```

Comment:
I think this is something we added only in 2.15 right? Then deleting should be fine

Reply:
@QilongTang well.... this property was added by Astul like a year ago, in the next pull request: https://github.com/DynamoDS/Dynamo/pull/11767 and currently is used in the Visibility property for the Security tab but now that is visible it has no use, unless we are planning to add another tab and hide it in debug mode. if that's the case we can keep it, please let me know if we should keep it. Thanks

Reply:
It's very unlikely 3rd party tools will depend on this flag and since this is on view level, I think this is OK to remove